spec/migrations/20160911130449_add_value_serialized_to_custom_attribute_spec.rb

```diff
@@ -0,0 +1,10 @@
+require_migration
+describe AddValueSerializedToCustomAttribute do
+  migration_context :up do
+    it 'migrate current values to serialized values correctly' do
+      FactoryGirl.create(:custom_attribute, :name => 'example', :value => "foo")
+      migrate
+      expect(YAML.load(CustomAttribute.find_by_name("example").serialized_value)).to eq("foo")
```

Comment:
Getting this when using the stub, undefined method `serialized_value' for #<ActiveRecord::Relation []>, it seems that the migrate action doesn't add the new column to the stub.